common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -193,6 +193,10 @@ def modulestore():
             settings.MODULESTORE['default'].get('OPTIONS', {})
         )
 
+        if settings.FEATURES.get('CUSTOM_COURSES_EDX'):
+            from ccx.modulestore import CCXModulestoreWrapper
```

Comment:
Yeah, a comment would be great. (Figuring out the circular import would be awesome too, but isn't required for right now).